.Rbuildignore

```diff
@@ -1,3 +1,4 @@
+^codecov\.yml$
```

Comment:
Hi @tdhock , I have added code coverage section. Let me know if everything seems fine so I can merge this branch with master.